docs/data-management/post-session.md

```diff
@@ -286,8 +286,19 @@ To support backward compatibility (and some extra, currently unsupported feature
     ```
 
 ### Convert eye-tracking into BIDS with *bidsphysio*
+- [ ] Pull the latest docker image of esavary/bidsphysio with:
+``` shell
+docker pull esavary/bidsphysio
+```
+- [ ] Open [schedule_ET_data.tsv](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the .EDF file corresponding to the session you want to convert.
+- [ ] In the folder containing your data, create a new folder named metadata. Copy the file containing information about the eye-tracker, named [info_ET.json](../code/physioconv/info_ET.json), inside this metadata folder.
```

Comment:
Please start every sentence in a new line. Beyond the stylistic comment, I cannot understand this. Could you rephrase?
```suggestion
- [ ] In the folder containing your data, create a new folder named metadata.
    Copy the file containing information about the ET, named [`info_ET.json`](../code/physioconv/info_ET.json), inside this metadata folder.
```